staging/src/k8s.io/client-go/tools/auth/exec/exec.go

```diff
@@ -0,0 +1,130 @@
+package exec
```

Comment:
How does a kubernetes developer find out about helper packages such as this one? Is there some sort of documentation that needs to be updated (besides the website)?

Reply:
Partially answered by https://github.com/kubernetes/kubernetes/pull/95489/files#r510141353. Resolving.